frontend/src/routes/Presentations.svelte

```diff
@@ -1,69 +1,37 @@
 <script lang="ts">
+
   // api imports
-  import { getCall } from "../api/presentationAxios";
+  import { getPresentations } from "../api/presentationAxios";
   //component imports
   import PresentationTableData from "../lib/PresentationTableFormat.svelte"
+  import SchemaBuilder from '../lib/SchemaBuilder.svelte'
 
-  interface PresentationTableInterface {
-      id:number;
-      name:string;
-      jobTitle:string;
-      email:string;
-      role:string;
-    }
-  export const testPresentationData: PresentationTableInterface[] = [
-    {
-        id:0,
-        name: "Jane Cooper",
-        jobTitle: "Regional Paradigm Technician",
-        email:"jane.cooper@example.com",
-        role: "Admin",
-      },
-      {
-        id:1,
-        name: "Cody Fisher",
-        jobTitle: "Product Directives Officer",
-        email:"cody.fisher@example.com",
-        role: "Owner",
-      },
-      {
-        id:2,
-        name: "Bob McBob",
-        jobTitle: "Customer service",
-        email:"BobMcBob@example.com",
-        role: "Employee",
-      },
-  ]
-const mockendUrl = 'https://mockend.com/aviarytech/wacinsteins-monster/users'
-async function apiGetCall():Promise<any>{
-      let res = await getCall(mockendUrl)
-      console.log(res)
-      return res
+  async function apiGetCall():Promise<any>{
+    let res = await getPresentations()
+    console.log(res)
+    return res
   }
   let displayData = false
-  let data:PresentationTableInterface[] = []
+  let data:Array<string> = []
```

Comment:
this shouldn't be an Array type it should be a list of string type
```suggestion
  let data: string[] = []
```

Reply:
and all other similar instances

Reply:
sounds good, I'll pay more attention to Array<t> that I infer. I often use them interchangeably. Made changes